master/internal/api_experiment.go

```diff
@@ -99,7 +99,7 @@ func (a *apiServer) GetExperiment(
 		JobSummary: &jobv1.JobSummary{},
 	}
 
-	if model.TerminalStates[model.StateFromProto(exp.State)] {
+	if model.ActiveState != model.StateFromProto(exp.State) {
```

Comment:
maybe I should've taken out the part to fetch jobsummary entirely to demo. my expectation is that that would result in the same behavior.

Reply:
this condition makes more sense to me.